### ISSUE TITLE
Color contrast of 2.37:1 for EDITOR'S NOTE: on WCAG 2.1

### ISSUE THREAD

**Issue by goodwitch** [w3c/wcag#313]
Dear W3C,  the green (#22bb22) "EDITOR'S NOTE" text on light green (#e9fbe9) does not meet color contrast requirements (on this page and others https://www.w3.org/TR/WCAG21/ )

aXe reports: 
Element has insufficient color contrast of 2.37 (foreground color: #22bb22, background color: #e9fbe9, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Comment by fstrr** [comment]
Closing this issue as it appears to be resolved. The contrast of the Editor's Note heading text in the [Vocabulary Structure and Implementations section](https://www.w3.org/TR/personalization-semantics-content-1.0/#vocabulary-structure-and-implementations) of the Personalization Semantics Content Module has a contrast of 4.58:1.
